packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -1154,6 +1158,16 @@ class SliverAppBar extends StatefulWidget {
   /// Defaults to [NavigationToolbar.kMiddleSpacing].
   final double titleSpacing;
 
+  /// The size of the app bar when it is collapsed.
```

Comment:
This property is no longer just the size of the app bar ... It would be OK to say: Defines the height of the app bar when it is collapsed.